chainercv/links/model/ssd/transforms.py

```diff
@@ -66,46 +66,52 @@ def convert(img, alpha=1, beta=0):
         img[img > 255] = 255
         return img.astype(np.uint8)
 
-    def brightness(cv_img):
+    def brightness(cv_img, brightness_delta):
```

Comment:
`delta` is enough unique in the function `brightness`. Can you rename `brightness_delta` -> `delta`?